modules/ROOT/pages/errors/gql-errors/index.adoc

```diff
@@ -103,6 +103,19 @@ Status description:: error: connection exception - general driver transient erro
 
 Status description:: error: connection exception - general driver database error. Remote execution failed with message `{ <<msg>> }`.
 
+=== xref:errors/gql-errors/08N19.adoc[08N19]
+
+Status description:: error: connection exception - shard execution transient error. Communication with shard `{ <<graph>> }` failed. with message `{ <<msg>> }`.
```

Comment:
This is what it says in the codebase, but I am a bit sceptical about the sentence with a dot followed by a lowercase `with` in this and the next two errors. Perhaps we can make a card for composite about changing this in codebase and docs later